README.md

```diff
@@ -0,0 +1,119 @@
+
+
+# Cisco IOSXR Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/cisco.iosxr) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/cisco.iosxr)-->
+
+The Ansible Cisco IOSXR collection includes a variety of Ansible content to help automate the management of Cisco IOSXR network appliances.
+
+This collection has been tested against Cisco IOSXR version 6.1.3..
+
+### Supported connections
+The Cisco IOSXR collection supports ``network_cli``  and ``netconf`` connections.
+
+## Included content
+
+Click the ``Content`` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the Cisco IOSXR collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install cisco.iosxr
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: cisco.iosxr
+    version: 0.0.3
+```
+## Using this collection
+
+
+This collection includes [network resource modules](https://docs.ansible.com/ansible/latest/network/user_guide/network_resource_modules.html).
+
+### Using modules from the Cisco IOSXR collection in your playbooks
+
+You can call modules by their Fully Qualified Collection Namespace (FQCN), such as `cisco.iosxr.iosxr_l2_interfaces`.
+The following example task replaces configuration changes in the existing configuration on a Cisco IOSXR network device, using the FQCN:
+
+```yaml
+---
+ - name: Replace device configuration of specified L2 interfaces with provided configuration.
+   cisco.iosxr.iosxr_l2_interfaces:
+     config:
+       - name: GigabitEthernet0/0/0/4
+         native_vlan: 40
+         l2transport: True
+         l2protocol:
+         - stp: forward
+       - name: GigabitEthernet0/0/0/3.900
+         q_vlan:
+         - 20
+         - any
+     state: replaced
+
+```
+
+Alternately, you can call modules by their short name if you list the `cisco.iosxr` collection in the playbook's `collections`, as follows:
+
+```yaml
+---
+- hosts: iosxr01
+  gather_facts: false
+  connection: network_cli
```

Comment:
when FQCN is used in task name 'collections' keyword is not required.

Reply:
not sure what this comment refers to here?  This example is specifically that - includes `collections` keyword and does not use FQCN.

Reply:
Oh yeah, you are right, I made a mistake here.